### DISCUSSION TITLE
Request for full support of Chakra UI Svelte - A port of Chakra UI for Svelte

### DISCUSSION THREAD

**Discussion by elcharitas** [chakra-ui/chakra-ui#7462] [Show and tell]
### Description

Chakra UI has been a game-changer for React and Vue developers, and I believe that Svelte developers can also benefit from its easy-to-use components, accessibility, and customization options.

### Problem Statement/Justification

Chakra UI has been a widely adopted UI library for React/Vue developers due to its ease of use, accessibility, and customization options. However, for developers using Svelte, there is currently no official port of Chakra UI available. As a result, Svelte developers have to resort to other UI libraries, which may not offer the same level of accessibility and customization options as Chakra UI.

Thus, I set out to fill this gap by providing a port of Chakra UI for Svelte developers and this is [Chakra UI Svelte](https://github.com/elcharitas/chakra-ui-svelte). Svelte developers can now benefit from the same ease of use, accessibility, and customization options as React developers, allowing them to build consistent, accessible, and responsive user interfaces. Additionally, with the integration of SvelteKit, developers would be able to leverage the power of server-side rendering to improve their web application's performance and accessibility.

### Proposed Solution or API

[Chakra UI Svelte](https://github.com/elcharitas/chakra-ui-svelte) is a UI library for building responsive and accessible user interfaces in Svelte. It provides a comprehensive set of customizable components based on Chakra UI design principles, with an intuitive API and seamless integration with [SvelteKit](https://github.com/sveltejs/kit). Check out the project at https://github.com/elcharitas/chakra-ui-svelte to start building accessible and responsive web applications with ease.

### Additional Information

A preview is available at https://chakra-ui-svelte.vercel.app

**Comment by segunadebayo** [comment]
This looks great @elcharitas.

Love to see that you took the initiative to make this happen. I'll keep an eye out for the progress of the library.

**Reply by vigneshksaithal** [reply]
Any updates on this?
